examples/touch.irp.c

```diff
@@ -0,0 +1,15 @@
+void provide_a(){
+  int a;
+  a = 10;
+}
+
+void provide_b(){
+  a = 20;
+  touch_a();
+  int b;
+  b = a;
+}
+
+int main(){
+  printf("b = %i", b);
+}
```

Comment:
May we should do A -> B -> C where C touch A and use b?
So we will need to call `provide_b` after the `touch_a()`, and our test will be more robust (testing more things).
Or we can have two tests. More test is better :)